### PR TITLE
Fix the debug view detection

### PR DESCRIPTION
**Pull request by sbrunner**
With the previous version if we have:
debug_url_prefix = '_debug'
req.path_info = '_debug_toolbar'

It won't work...

### CODE REVIEW

**Comment by sbrunner** [comment]
Fix  #13

**Comment by mmerickel** [comment]
LGTM

**Comment by sbrunner** [comment]
Thanks :-)

File: weberror/evalexception.py [hunk 1]
```diff
@@ -254,13 +254,14 @@ def __call__(self, environ, start_response):
         environ['weberror.evalexception'] = environ['paste.evalexception'] = \
             self
         req = Request(environ)
-        if req.path_info.startswith('/'+self.debug_url_prefix):
+        req_path = req.path_info.split('/')[1:len(self.debug_url_prefix + 1)]
```
Comment (ehberger) [inline weberror/evalexception.py]: This line is incorrect.  It attempts to add 1 to an array, and crashes.  The "+1" should be outside of the len() call.

Comment (mmerickel) [inline weberror/evalexception.py]: Can you submit a PR to fix this?

**Pull request merged**